### ISSUE TITLE
Model required=true attribute no generating not-null contraint on Mysql

### ISSUE THREAD

**Issue by francoisgrogor** [balderdashy/sails-mysql#116]
Hi All,

I defined sails model with an attribute with the "required : true" parameter.

When I lift sails, it generates the tables fine, but the fields in question do not contain the not null contraint. Also, as a side not I dont see the FK contraints either?

Sails version :  0.10-rc7
sails-mysql version : 0.10.0-rc7
Mysql version : mysql  Ver 14.14 Distrib 5.6.17, for osx10.8 (x86_64)

Any thoughts? 

Model : 

module.exports = {

    schema : true,

    attributes: {
        name : {
            type : 'STRING',
            maxLength: 50,
            required: true
        },
        description : {
            type : 'STRING',
            maxLength: 200,
            required: true
        },
        products: {
            collection: 'Product',
            via: 'events'
        }

    }

**Comment by sapronlee** [comment]
`required` replace `notNull`

**Comment by francoisgrogor** [comment]
Hi Sapronlee, Will these attribute result in the constraints being created in at the database level (Mysql)?

**Comment by sapronlee** [comment]
@francoisgrogor yes
my `package.json` is
`"sails": "~0.11.0"
"sails-mysql": "^0.10.12"`

my `User.js` is
`attributes: {
    email: {
      type: 'string',
      size: 100,
      notNull: true,
      unique: true
    }
}`

this `notNull = true` attribute result make `email not null` in at the database mysql

**Comment by sailsbot** [comment]
Thanks for posting, @francoisgrogor.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
